src/components/Modal.spec.js

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from '@testing-library/react';
+import Modal from './Modal';
+import userEvent from '@testing-library/user-event';
+
+describe('Modal', () => {
+  it('renders a warning message and two buttons when clicking on the delete button', () => {
+    render(<Modal visible={true} />);
```

Comment:
Modal doesn't get a prop `visible`.
```suggestion
    render(<Modal />);
```